ci/templates/hosted-cover-jobs.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           poolName: "Azure Pipelines"
           os: Linux
           jdk: "1.11"
-          testset: "coverage-cairo"
```

Comment:
what necessitated CI change?

Reply:
![image](https://github.com/user-attachments/assets/48b9c834-1ca4-4d2e-843b-9dedc2359535)
 
`coverage-cairo` was used twice, that overwrote the same report by the last finishing job. That resulted in false 0 coverage for the PR change